packages/storybook/stories/buttons.stories.mdx

```diff
@@ -1,12 +1,13 @@
 import { Meta, Canvas } from '@storybook/addon-docs/blocks';
+import './buttons.style.scss';
 
 <Meta title="Components/Buttons" />
 
-## Default buttons
+## Primary buttons
 
 <Canvas>
-  <Story name="Default">
-    <div>
+  <Story name="Primary">
```

Comment:
This `Story` div doesn't render in iframes on design.va.gov

Reply:
I know it shows up in Storybook, but it doesn't in the iframe 😢

Reply:
Oh good catch - I didn't check it in the iframe.